aries-backchannels/python/storage.py

```diff
@@ -85,8 +102,12 @@ def pop_resource_latest(data_type):
         data_id = data_ids[len(data_ids) - 1]
         if data_type in storage[data_id]:
             if 0 < len(storage[data_id][data_type]):
-                data = storage[data_id][data_type][0]
-                del storage[data_id][data_type][0]
+                if len(storage[data_id][data_type]) > 1:
+                    data = storage[data_id][data_type][len(storage[data_id][data_type])-1]
```

Comment:
Tip: You can just use `-1` instead of `len(storage[data_id][data_type])-1` to get the last item of an list

Reply:
Thanks for that. Will try.